lib/tokenizer.ts

```diff
@@ -154,6 +168,8 @@ export const tokenizer = (regexpStr: string): Root => {
           }
 
           group.remember = false;
+        } else {
+          groupCount += 1;
```

Comment:
oh i hadn't realized they can't be referenced if they're special group types. cool